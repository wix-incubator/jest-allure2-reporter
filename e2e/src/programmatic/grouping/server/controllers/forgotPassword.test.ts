import { $Epic, $Feature, $Story, $Tag } from 'jest-allure2-reporter';

$Tag('server');
$Epic('Authentication');
$Feature('Restore account');
describe('POST /forgot-password', () => {
  $Story('Validation');
  it('should return 401 if user is not found', () => {
    // ...
  });

  $Story('Happy path');
  it('should return 200 if reset link is sent successfully', () => {
    // ...
  });

  $Story('Error handling');
  it('should handle server errors gracefully', () => {
    // ...
  });
});
