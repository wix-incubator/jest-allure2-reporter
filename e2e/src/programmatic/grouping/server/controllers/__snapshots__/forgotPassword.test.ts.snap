// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`POST /forgot-password should return 401 if user is not found 1`] = `
{
  "code": 401,
  "seed": 0.06426173461501827,
}
`;
